cli/src/main/scala/aqua/AquaError.scala

```diff
@@ -10,13 +10,14 @@ sealed trait AquaError {
   def showForConsole(script: String): String
 }
 
-case class CustomSyntaxError(span: Span, message: String) extends AquaError {
+case class CustomSyntaxError(name: String, span: Span, message: String) extends AquaError {
```

Comment:
Here you added filename-awareness to a non-filename-aware error type: `Span.F` can be applied to any text input, and errors capture only the offset, `FileSpan.F` can be used only for file input, and augments `Span` with the filename.

I'm not sure it's a good design.